library/ai/chat-memory/forage-memory-infinispan/src/main/java/org/apache/camel/forage/memory/chat/infinispan/InfinispanConfigEntries.java

```diff
@@ -7,26 +7,117 @@
 import org.apache.camel.forage.core.util.config.ConfigEntries;
 import org.apache.camel.forage.core.util.config.ConfigEntry;
 import org.apache.camel.forage.core.util.config.ConfigModule;
+import org.apache.camel.forage.core.util.config.ConfigTag;
 
 public final class InfinispanConfigEntries extends ConfigEntries {
-    public static final ConfigModule SERVER_LIST = ConfigModule.of(InfinispanConfig.class, "infinispan.server-list");
-    public static final ConfigModule CACHE_NAME = ConfigModule.of(InfinispanConfig.class, "infinispan.cache-name");
-    public static final ConfigModule USERNAME = ConfigModule.of(InfinispanConfig.class, "infinispan.username");
-    public static final ConfigModule PASSWORD = ConfigModule.of(InfinispanConfig.class, "infinispan.password");
-    public static final ConfigModule REALM = ConfigModule.of(InfinispanConfig.class, "infinispan.realm");
-    public static final ConfigModule SASL_MECHANISM =
-            ConfigModule.of(InfinispanConfig.class, "infinispan.sasl-mechanism");
-    public static final ConfigModule CONNECTION_TIMEOUT =
-            ConfigModule.of(InfinispanConfig.class, "infinispan.connection-timeout");
-    public static final ConfigModule SOCKET_TIMEOUT =
-            ConfigModule.of(InfinispanConfig.class, "infinispan.socket-timeout");
-    public static final ConfigModule MAX_RETRIES = ConfigModule.of(InfinispanConfig.class, "infinispan.max-retries");
-    public static final ConfigModule POOL_MAX_ACTIVE =
-            ConfigModule.of(InfinispanConfig.class, "infinispan.pool.max-active");
-    public static final ConfigModule POOL_MIN_IDLE =
-            ConfigModule.of(InfinispanConfig.class, "infinispan.pool.min-idle");
-    public static final ConfigModule POOL_MAX_WAIT =
-            ConfigModule.of(InfinispanConfig.class, "infinispan.pool.max-wait");
+    public static final ConfigModule SERVER_LIST = ConfigModule.of(
+            InfinispanConfig.class,
+            "infinispan.server-list",
+            "Comma-separated list of Infinispan server addresses in format 'host1:port1,host2:port2'",
+            "Server List",
+            "localhost:11222",
+            "string",
+            true,
+            ConfigTag.COMMON);
```

Comment:
Once we get more help, one potential cleanup we might want to do is to move these to a properties file (internal to forage). Something like:

```properties
config.entry.server-list.description=<description>
config.entry.server-list.label=<label>
...
```

Then we load it on the ConfigModule i.e.: `ConfigModule.of(class, <modulename>, "server-list", "description")`